www/css/index.css

```diff
@@ -52,9 +52,10 @@ header {
    width: 100%;
    height: var(--header-height);
    background: var(--background);
-   background-image: '';
-   background-size: contain;
+   background-image: url('../resources/background.jpg');
```

Comment:
I saw there is also a background.png. Any reason for that?

Reply:
Removed png.